extensions/vertx-common/runtime/src/main/java/io/quarkus/vertx/common/runtime/VertxConfiguration.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.vertx.runtime;
+package io.quarkus.vertx.common.runtime;
```

Comment:
Wondering if should not move all configuration-related classes in a `config` package.

Reply:
Let's do that in another PR if we think it's a good idea. I don't have a strong opinion about that but maybe that would be more readable.